template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/LoginWith2fa.cshtml.cs

```diff
@@ -77,17 +77,17 @@ public class InputModel
 
             if (result.Succeeded)
             {
-                _logger.LogInformation("{UserName} logged in with 2fa.", user.UserName);
+                _logger.LogInformation("User with id '{UserId}' logged in with 2fa.", user.Id);
```

Comment:
The general rationale is that directly accessing the properties on IdentityUser tie you to the default POCO/implementation, making it harder for apps to switch to a different store implementation that might not expose Id like this.  Leaky abstraction...